README.md

```diff
@@ -14,169 +12,202 @@
+speeds with commodity infrastructure and far less complexity than today.
+SnappyData fulfills this promise by
+- Enabling streaming, transactions and interactive analytics in a single unifying system rather than stitching different
+solutions—and
+- Delivering true interactive speeds via a state-of-the-art approximate query engine that leverages a multitude of
+synopses as well as the full dataset. SnappyData implements this by deeply integrating an in-memory database into
+Apache Spark.
 
 ### Purpose
-Here we use a simplified Ad Analytics example, which streams in [AdImpression](https://en.wikipedia.org/wiki/Impression_(online_media)) logs, pre-aggregating the logs and ingesting into the built-in in-memory columnar store (where the data is stored both in 'exact' form as well as a stratified sample).
+Here we use a simplified Ad Analytics example, which streams in [AdImpression](https://en.wikipedia.org/wiki/Impression_(online_media))
+logs, pre-aggregating the logs and ingesting into the built-in in-memory columnar store (where the data is stored both
+in 'exact' form as well as a stratified sample).
 We showcase the following aspects of this unified cluster:
-- Simplicity of using SQL or the [DataFrame API](http://spark.apache.org/docs/latest/sql-programming-guide.html#dataframes) to model streams in spark.
-- The use of SQL/SchemaDStream API (as continuous queries) to pre-aggregate AdImpression logs (it is faster and much more convenient to incorporate more complex analytics, rather than using map-reduce).
-- Demonstrate storing the pre-aggregated logs into the SnappyData columnar store with high efficiency. While the store itself provides a rich set of features like hybrid row+column store, eager replication, WAN replicas, HA, choice of memory-only, HDFS, native disk persistence, eviction, etc we only work with a column table in this simple example.
-- Run OLAP queries from any SQL client both on the full data set as well as sampled data (showcasing sub-second interactive query speeds). The stratified sample allows us to manage an infinitely growing data set at a fraction of the cost otherwise required.
+- Simplicity of using the [DataFrame API](http://spark.apache.org/docs/latest/sql-programming-guide.html#dataframes) to
+model streams in spark.
+- The use of Structured Streaming API to pre-aggregate AdImpression logs (it is faster and much more convenient to
+incorporate more complex analytics, rather than using map-reduce).
+- Demonstrate storing the pre-aggregated logs into the SnappyData columnar store with high efficiency. While the store
+itself provides a rich set of features like hybrid row+column store, eager replication, WAN replicas, HA, choice of memory-only, HDFS, native disk persistence, eviction, etc we only work with a column table in this simple example.
+- Run OLAP queries from any SQL client both on the full data set as well as sampled data (showcasing sub-second
```

Comment:
Should a link to the AQP documentation be provided here?

Reply:
These are just formatting changes. In current master also AQP is not mentioned. Although down the line we have explained the AQP capabilities separately in the same readme file.